src/core/network/qgsnetworkaccessmanager.h

```diff
@@ -570,6 +570,96 @@ class CORE_EXPORT QgsNetworkAccessManager : public QNetworkAccessManager
     % End
 #endif
 
+    /**
+     * Sets an advanced request pre-processor function, which allows manipulation of a network request before it is processed.
+     *
+     * The \a processor function takes the QNetworkRequest, network operation, and request data as its arguments, and can mutate the request if necessary.
```

Comment:
what is a "network operation" exacty ? Is it an enumeration ?

Reply:
Docs expanded